docs/google-kubernetes-tutorial.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Deploy TiDB to Kubernetes on Google Cloud
+summary: Tutorial for deploying TiDB on Google Cloud using Kubernetes.
+category: operations
+---
+
+# Deploy TiDB, a distributed MySQL compatible database, to Kubernetes on Google Cloud
+
+## Introduction
+
+This tutorial is designed to be [run in Google Cloud Shell](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/pingcap/tidb-operator).  It takes you through these steps:
+
+- Launching a new 3 node Kubernetes cluster (optional)
+- Installing the Helm package manager for Kubernetes
+- Deploying the TiDB Operator
+- Deploying your first TiDB cluster
+- Connecting to TiDB
+- Shutting down down the Kubernetes cluster (optional)
+
+## Select a Project
+
+Please select a project before proceeding.  The approximate cost for running the compute resources in this tutorial is 10 cents/hour.
+
+<walkthrough-project-billing-setup key="project-id">
+</walkthrough-project-billing-setup>
+
+## Enable API Acess
+
+This tutorial will require use of the Compute and Container APIs.  Please enable them before proceeding:
+
+<walkthrough-enable-apis apis="container.googleapis.com,compute.googleapis.com">
```

Comment:
I guess this is a link to GKE documentation of how to enable APIs. But it doesn't render as a link on GitHub.

Reply:
Yes, it is called a [directive](https://cloud.google.com/shell/docs/tutorial-markdown-reference).  In the Cloud Shell the user is prompted.

There is one other directive before this, which is to create or select a project.